src/Card/DetailCard/DetailCard.tsx

```diff
@@ -30,57 +30,57 @@ function DetailCard({
 }: DetailCardProps) {
   // render the detail card
   return (
-    <Fragment>
-      <Stack
-        mt={1}
-        mb={3}
+    <Stack
+      id="DetailCard"
+      mt={1}
+      mb={3}
+      sx={{
+        display: "flex",
+        flexDirection: "column",
+        flexGrow: 1,
+        height: "100%",
+        width
+      }}
+    >
+      <DetailCardHeader
+        title={title}
+        subtitle={subtitle}
+        buttonsStack={buttonsStack}
+        labels={labels}
+        width={width}
+        onClickLabel={onClickLabel}
+      />
+      <Box
         sx={{
           display: "flex",
-          flexDirection: "column",
-          flexGrow: 1,
+          flexDirection: "row",
           height: "100%",
-          width
+          my: 2,
+          overflow: "auto"
         }}
       >
-        <DetailCardHeader
-          title={title}
-          subtitle={subtitle}
-          buttonsStack={buttonsStack}
-          labels={labels}
-          width={width}
-          onClickLabel={onClickLabel}
-        />
-        <Box
-          sx={{
-            display: "flex",
-            flexDirection: "row",
-            height: "100%",
-            my: 2,
-            overflow: "auto"
-          }}
-        >
-          <Box ml={0.5}>
-            <FileCard
-              media={media}
-              width={368}
+        <Box ml={0.5}>
+          <Card>
+            <Infographic media={media} />
```

Comment:
I think we can integrate the `Infographic` in `SummaryCard` component as well.

Reply:
This is done in #896